### ISSUE TITLE
modifyAll and setTo

### ISSUE THREAD

**Issue by justinhj** [adamw/quicklens#27]
Is it possible to use 'modifyAll' and something like 'setTo' to set multiple fields without doing duplicate copy operations? In the example code I only see modifyAll being used with 'using'. 

Ideally setTo would take a tuple that matches the order of arguments to modifyAll

**Comment by adamw** [comment]
Wouldn't a chain:

```
person
  .modify(_.f1).setTo(v1)
  .modify(_.f2).setTo(v2)
  .modify(_.f3).setTo(v3)
```

be more readable?
With the tuples approach, you have to mentally connect the fields-tuple with the values-tuple, which could be error prone?

**Comment by justinhj** [comment]
Yes you're right it's more readable and safer that way. I was just wondering if it would be more efficient if it was done in one call.

**Comment by adamw** [comment]
Probably, a bit, but then if modifying a case class once or three times makes a difference, maybe it's not a good idea to use (immutable) case classes anyway ;)

**Comment by justinhj** [comment]
That's true. We're making heavy use of this pattern but I can always refactor it locally if there's a hot point in the code. 
Thanks for the input.
